app/handlers/configHomography.py

```diff
@@ -31,43 +28,25 @@ class UploadHomographyHandler(BaseHandler):
     """
 
     def prepare(self):
-        identifier = self.get_body_argument("identifier")
-        if StatusHelper.get_status(identifier)[Status.Type.UPLOAD_HOMOGRAPHY] == Status.Flag.IN_PROGRESS:
+        self.identifier = self.get_body_argument("identifier")
+        if StatusHelper.get_status(identifier)[Status.Type.CONFIG_HOMOGRAPHY] == Status.Flag.IN_PROGRESS:
             status_code = 423
             self.error_message = "Currently uploading homography. Please wait."
             raise tornado.web.HTTPError(status_code = status_code)
-        StatusHelper.set_status(identifier, Status.Type.UPLOAD_HOMOGRAPHY, Status.Flag.IN_PROGRESS)
-
-
-    def initialize(self):
-        # Make sure the BaseHandler is initialized
-        super(UploadHomographyHandler, self).initialize()
-
-        self.identifier = None
-        self.files = {}
+        StatusHelper.set_status(identifier, Status.Type.CONFIG_HOMOGRAPHY, Status.Flag.IN_PROGRESS)
 
     def post(self):
-        self.files = self.request.files
-        self.identifier = self.get_body_argument('identifier')
         self.up_ratio = float(self.get_body_argument('unit_pixel_ratio'))
         self.write_homography_files()
-        StatusHelper.set_status(self.identifier, Status.Type.UPLOAD_HOMOGRAPHY, Status.Flag.COMPLETE)
+        StatusHelper.set_status(self.identifier, Status.Type.CONFIG_HOMOGRAPHY, Status.Flag.COMPLETE)
         self.finish("Upload Homography")
 
     def write_homography_files(self):
         project_dir = get_project_path(self.identifier)
         aerial_pts = self.get_body_argument('aerial_pts')
         camera_pts = self.get_body_argument('camera_pts')
+
         homography, mask = cv2.findHomography(\
                             np.array(literal_eval(camera_pts)),\
```

Comment:
Yup want to make sure we don't have any issues here :) Looking at the documentation, this actually might need to be try/catched to make sure we grab any malformed data exceptions.